docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+<http://avocado-framework.readthedocs.io/en/52.0/>`_
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  `avocado.core.plugin_interfaces` module.
+- A contrib script for running `kvm-unit-tests`.
+- Support for running tests in Docker container.
+- Introduction of the “Fail Fast” feature.
+- Special keyword `latest` for replaying previous jobs.
+- The introduction of a tool that generated a diff-like report of two
+  jobs.
+- New `avocado.utils.process.SubProcess.get_pid()` method.
+- The introduction of a `rr` based wrapper.
+- The `coredump` generation contrib scripts will check if the user
+  running Avocado is privileged to actually generate those dumps.
+- New `avocado.utils.linux_modules.check_kernel_config()`.
+- The automatic VM IP detection that kicks in when one uses `--vm-domain`
+  without a matching `--vm-hostname`, now uses a more reliable method
+  (libvirt/qemu-gust-agent query).
+- Set `LC_ALL=C` by default on sysinfo collection to simplify avocado
+  diff comparison between different machines.
+- List result plugins (JSON, XUnit, HTML) in avocado plugins command
+  output.
+- Multiplexer: it now defines an API to inject and merge data into the
+  multiplexer tree.
+- It’s now possible to disable plugins by using the configuration file.
+- Users of the `avocado.utils.process` module will now be able to access
+  the process ID in the `avocado.utils.process.CmdResult`.
+- Improved `avocado.utils.iso9660` with a more complete standard API
+  across all back-end implementations.
+- Improved `avocado.utils.build.make()`, which will now return the make
+  process exit status code.
+- Better error handling of the virtual machine plugin (`--vm-domain` and
+  related options).
+- The `avocado.utils.process` library makes it possible to ignore
+  background processes that never finish.
+- When discovering tests on a directory, the result now is a properly
+  alphabetically ordered list of tests.
+- Plugins can now be setup to run at a specific order.
+- A contrib script that looks for a job result directory based on a
+  partial (or complete) job ID is now available at
+  `contrib/scripts/avocado-get-job-results-dir.py`.
+- Support for filtering tests by user supplied “tags”.
+- Users can now choose to keep the complete set of files, including
+  temporary ones, created during an Avocado job run by using the
+  `--keep-tmp` option.
+- Tests running with the external runner (`--external-runner`) feature
+  will now have access to the extended behavior for SIMPLE tests, such
+  as being able to exit a test with the WARNING status.
+- Users will now be able to properly run tests based on any Unicode
+  string (as a test reference).
```

Comment:
Basically encoding support was improved so now Avocado should safely treat localized test-names.